### DISCUSSION TITLE
how to enable TinyMCE statusbar?

### DISCUSSION THREAD

**Discussion by atomGit** [GetPublii/Publii#1839] [Help]
the TinyMCE status bar was removed for some odd reason - this is aggravating since certain elements in the WYSIWYG editor *cannot be selected* without it (except by editing the HTML)

for example, add the HTML `<p>eee <code></code></p>` and then try selecting or deleting the empty code tags

i want the statusbar back, but this isn't working...

in `/Publii/config/tinymce.override.json` i have:

```
{
	"plugins": "statusbar autoresize"
}
```

and in `/Publii/themes/simple/config.json` i added the "extensions" code:

```
{
    "name": "Simple",
    [...]
        },
        "footerMenu": {
            [...]
        }
    },
    "extensions": {
            "postEditorConfigOverride": true,
            "postEditorCustomScript": true
     },
    [...]
```

doesn't work - status bar doesn't appear

**Comment by candidexmedia** [comment]
Do you have a screenshot of what the status bar looked like?

**Comment by atomGit** [comment]
it was a just a simple status bar like any other status bar - if the caret was on a 'p' (paragraph) element, then there would be a 'p' in it, etc., otherwise it was empty

https://www.tiny.cloud/images/legal/tinymce6-homepage-demo.png

**Reply by candidexmedia** [reply]
I did some digging, and it looks like it was [hidden with CSS](https://github.com/GetPublii/Publii/blob/0047d428cb2e6b4b8b8dcfbe580203aaaa39ab98/app/src/scss/editor/editor-overrides.scss#L46):

![image](https://github.com/user-attachments/assets/df6f62be-ec46-4335-8fcd-90eb0d4e4c07)

The selector is `.tox-statusbar__path`, and the CSS rule that worked in dev mode was as follows:

```
.tox-statusbar__path {
    display: flex !important;
}
```

![image](https://github.com/user-attachments/assets/651fa5aa-a485-433c-8657-ea7c3d342d91)


That said, I wasn't able to replicate it by adding the rule in `editor.css` ([based on the docs](https://getpublii.com/dev/post-editor-api/#customposteditorcssstyles)), so I gave up. Hopefully, the devs can chime in, or you can continue experimenting from there and find a solution.

---

> doesn't work - status bar doesn't appear

Based on Publii's [postEditor.config.js](https://github.com/GetPublii/Publii/blob/master/app/src/components/configs/postEditor.config.js), the statusbar was never set to "false" to hide it ([see Tiny docs](https://www.tiny.cloud/docs/tinymce/6/statusbar-configuration-options/#statusbar)), and statusbar isn't a plugin, which is why your method didn't work. If the statusbar _had_ been set to false, the way to override that feature would be to add the following to `tinymce.override.json:`

```
{
  "statusbar": true,
  "elementpath": true
}
```
